share/man/man5/style.mdoc.5

```diff
@@ -124,7 +124,8 @@ The
 .Fl width
 argument to the
 .Sy \&.Bl
-macro should match the length of the longest item in the list, e.g.:
+macro should match the length of the rendered first longest item in the list,
+e.g.:
```

Comment:
I'm sorry, but I don't understand the meaning/distinction from what is there already. Maybe this?

```suggestion
macro should match the (rendered) length of the longest item in the list,
e.g.:
```

Reply:
```suggestion
macro should match the length of the first longest item in the rendered list,
e.g.:
```
Maybe?

Reply:
Thanks for your patience everyone.